projects/core/src/lib/components/toolbar/streetview/streetview.component.css

```diff
@@ -0,0 +1,3 @@
+.streetview button {
```

Comment:
Is this needed? Should be covered by the `map-control-button-container` class already?

Reply:
indeed, not needed